libs/design-system-core/src/components/List/list-element.tsx

```diff
@@ -9,15 +9,14 @@ export type ListItemProps = { customStyle?: string } & ListItem;
 
 const ListElement: React.FC<ListItemProps> = props => {
   const { label, icon, color, disabled, customStyle = '', onClick, ...rest } = props;
+
+  const handleButtonClick = (e: React.SyntheticEvent) => {
+    e.stopPropagation();
```

Comment:
On a second thought, a component needs to use stopPropagation() a component further up the chain will break something. In this case need to fix the real issue rather than introducing a workaround. What do you think ?

Reply:
I added this because the list items were used in the menu which is popover like and could propagate clicks. Are you suggesting we define this stopPropagation() to the handlers themselves?

Reply:
It makes more sense to use stopPropagation() `onEntryFlag` and `onEntryRemove` inside EntryCard component.

Reply:
alright